### PR TITLE
Get info plist path action

### PR DESCRIPTION
**Pull request by SiarheiFedartsou**
#4895

### CODE REVIEW

**Comment by SiarheiFedartsou** [comment]
Hm, my PR seems to be not reason of test failure.

File: fastlane/docs/Actions.md [hunk 1]
```diff
@@ -1280,6 +1280,37 @@ You can also only receive the version number without modifying it
 version = get_version_number(xcodeproj: "Project.xcodeproj")
 ```
 
+### increment_version_number_in_plist
```
Comment (KrauseFx) [inline fastlane/docs/Actions.md]: How is this different from `increment_version_number`? That we don't use the Xcode feature to increment it?

Comment (KrauseFx) [inline fastlane/docs/Actions.md]: How is this different from `increment_version_number`? That we don't use the Xcode feature to increment it?

**Comment by KrauseFx** [comment]
Hey @SiarheiFedartsou, thanks for the pull request and sorry for the late reply. We just launched the fastlane plugin system, that allows you to build your own plugins without having to merge it into the main code base.

Get started here: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Plugins.md#create-your-own-plugin
You can basically copy over the action you already built after running `fastlane new_plugin` 🚀 

Please let us know how that works for you and if there is any way we can help 👍

**Comment by SiarheiFedartsou** [comment]
Just created a new plugin https://github.com/SiarheiFedartsou/fastlane-plugin-versioning . Thanks to everyone!

**Pull request closed without merging**